sdk/core/ci.yml

```diff
@@ -57,3 +57,6 @@ stages:
         - name: azure-core-tracing-opentelemetry
           groupId: com.azure
           safeName: azurecoretracingopentelemetry
+      AdditionalModules:
+        - name: azure-data-appconfiguration
```

Comment:
I'm not sure I fully understand why we need to add this here. Doesn't appconfig depend on Core and thus the `-am` should cause it to build?